config.json

```diff
@@ -931,7 +931,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Strong Static Typing",
+      "content": "Modern statically typed system with type inference, tuples, generics and sum types.",
+      "icon": "statically-typed"
+    },
+    {
+      "title": "Fast",
+      "content": "Compiles to C/C++/ObjC with no run time or virtual machine for blazing fast results.",
```

Comment:
There's a possible misreading of "with no **runtime or virtual machine for blazing fast results**" -> "with no blazing fast results" -> "it's slow". Also some potential confusion due to "with no virtual machine", since we have the Nim VM at compile time.

Maybe something like this?

> "Compiles to a small, blazing-fast binary (no runtime or virtual machine overhead)".

I think it's better to mention "small binary" (which many people care about, even outside embedded - it's also an advantage over Go) than mentioning C++ and ObjC. But this wording does lose out on "compiles to C" (and JavaScript). Can we find a wording that conveys everything?

Reply:
How about

> "Compiles using C/JS to a small, blazing-fast binary (no runtime or virtual machine overhead)."

Reply:
The problem is that compiling to JS isn't compiling to a binary :)

Reply:
How about this?

> "Compiles using C/C++/Objc to a small, blazing-fast binary (no runtime or virtual machine overhead)."

I think we leave out JS for now

Reply:
Bump